Apps/W1/HybridGP/app/src/pages/HybridGPErrorsOverviewFb.page.al

```diff
@@ -6,10 +6,6 @@ page 40132 "Hybrid GP Errors Overview Fb"
 {
     Caption = 'GP Upgrade Errors';
     PageType = CardPart;
-    InsertAllowed = false;
-    DelayedInsert = false;
-    ModifyAllowed = false;
-    SourceTable = "GP Migration Error Overview";
```

Comment:
This is necessary to get the Factbox counts to actually work when there are no records in the Error Overview table... Weird quirk of BC I found and confirmed with Nikola.